Makefile

```diff
@@ -91,11 +92,11 @@ test/e2e/testrunner-kind:
 # test/e2e/testrunner-kind.
 # After the tests run the cluster is deleted.
 # If you need a unique cluster name override CLUSTER_NAME.
-.PHONY: test/e2e/kind
```

Comment:
Anyone know if we can do a .PHONY while using %?

Reply:
@chrisseto ??

Reply:
Seems like no?

https://stackoverflow.com/questions/65664156/how-can-you-make-a-wildcard-make-rule-phony

Reply:
thats a bummer. lets go back to the 70s and yell at Stuart Feldman at Bell Labs.